azkaban-common/src/main/java/azkaban/jobExecutor/AbstractProcessJob.java

```diff
@@ -58,7 +61,7 @@ protected AbstractProcessJob(final String jobId, final Props sysProps, final Pro
     this.jobProps = jobProps;
     this.sysProps = sysProps;
     this.cwd = getWorkingDirectory();
-    this.jobPath = this.cwd;
+    this._jobPath = this.cwd;
   }
 
   private File createOutputPropsFile(final String id, final String workingDir) {
```

Comment:
Please change back to public for now, until we can verify no one is calling directly.

Reply:
copied.

Reply:
sorry. it was protected instead of public.